web/backend/controller/matchController.ts

```diff
@@ -0,0 +1,25 @@
+import { Matches } from "../schemas/matches.ts";
+import { Context } from "../../../deps/web/deps.ts";
+
+// creates a new participant with data in request
```

Comment:
Admin does not create matches. They are computed by the csp solver.
He has to create MatchBoxes and Matching Nights.

Reply:
Function is to safe the matches into the mongodb.
It can also be used in an automated way (or am i wrong?).